pd/src/components/staking.rs

```diff
@@ -607,12 +589,10 @@ impl Component for Staking {
                 }
             } else {
                 // This is a new validator definition.
-                validator_definitions.push(v.clone());
                 continue;
             }
 
-            // the validator definition has now passed all verification checks, so add it to the list
-            validator_definitions.push(v.clone());
+            // the validator definition has now passed all verification checks
         }
```

Comment:
Hmm, what's going on here? From the warning it looks like we're iterating over an empty list?

Reply:
Which warning? I don't see one locally

Reply:
never mind, it was a stale comment